api/apps/api/test/project/update-project.fixtures.ts

```diff
@@ -40,21 +47,39 @@ export const getFixtures = async () => {
         .patch(`/api/v1/projects/${projectId}`)
         .set('Authorization', `Bearer ${token}`)
         .send({
-          name: 'Test updated',
+          name: updatedName,
         }),
+    WhenProjectIsUpdatedWithARunningExport: async () => {
+      projectChecker.addPendingExportForProject(projectId);
+
+      return request(app.getHttpServer())
+        .patch(`/api/v1/projects/${projectId}`)
+        .set('Authorization', `Bearer ${token}`)
+        .send({
+          name: updatedName,
+        })
+        .expect(HttpStatus.BAD_REQUEST);
+    },
     WhenProjectIsUpdatedAsNotIncludedUser: async () =>
       await request(app.getHttpServer())
         .patch(`/api/v1/projects/${projectId}`)
         .set('Authorization', `Bearer ${notIncludedUserToken}`)
         .send({
-          name: 'Test updated',
+          name: updatedName,
         }),
     ThenWhenReadingProjectItHasNewData: async () => {
       const projectData = await request(app.getHttpServer())
         .get(`/api/v1/projects/${projectId}`)
         .set('Authorization', `Bearer ${token}`);
 
-      expect(projectData.body.data.attributes.name).toBe('Test updated');
+      expect(projectData.body.data.attributes.name).toBe(updatedName);
+    },
+    ThenWhenReadingProjectItHasTheOriginalData: async () => {
```

Comment:
for clarity, I suggest to split this into another action (e.g. `WhenGettingProjectData`) and an assertion (e.g. `ThenProjectIsUnchanged`.

At the very least, for clarity, I'd suggest:

```suggestion
    ThenProjectIsUnchanged: async () => {
```